css/style.css

```diff
@@ -1,19 +1,189 @@
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
-
+/* --==================== general styles ====================-- */
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+h1, h2, .about-text{
+  font-weight: lighter;
+}
+
+a, u{
+  color: white;
+}
+ /* --==================== HEADER ====================-- */
+ .header_logo {
+    height: 40px;
+    float: left;
+    position: relative;
+    padding-left: 100px;
+  }
+
+ /* --==================== NAVBAR ====================-- */
+ 
+ .nav_icon-text{
+  font-weight: 450;
+  color:#6b6b6b;
+  padding: 15px;
+}
+ 
+ .nav_icon{
+  font-weight: 450;
+  color:#9c9a9a;
+  padding: 15px;
+ }
+
+ .nav_icon:hover{
+   color: #d65924;
+
+ }
+
+ .container{
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  margin-right: 60px;
+  list-style: none;
+ }
+
+ /*hamburger menu */
+ 
+
+/* hamburger menu end */
+
+ /* --==================== INTRO SECTION ====================-- */
+ .backimage {
+  background-image: url("../img/first-background.jpg");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  color:white;
+  padding: 250px 0;
+}
+
+.backimage-text, h1.backimage-text {
+    text-align: center;
+}
+
+
+ h1{
+  margin-bottom:1rem;
+  font-size: 3.5rem;
+}
+
+.about-text{
+  font-size: 1.5rem;
+
+}
+
+.intro-btn {
+   background-color: #d65924;
+   border: none;
+   font-size: 0.8rem;
+   margin-top: 20px;
+   padding: 13px 25px;
+   border-radius: 3px;
+ }
+ /* --==================== BODY SECTION ====================-- */
+ .article__thumbnail{
+   width: 120px;
+   height: auto;
+ }
+
+ .service{
+   padding-top: 45px;
+ }
+
+ .services {
+  text-align: center;
+ }
+
+ .columns{
+  display: flex;
+  justify-content: center; /* space around*/
+  padding-bottom: 40px;
+}
+
+.column{
+  padding: 0 60px;
+  margin: 0 5px;
+}
+.column-text{
+  font-size: 1.3rem;
+}
+
+h2 {
+  text-align: center;
+  font-size: 2rem;
+}
+ /* --==================== FEATURE SECTION ====================-- */
+ .feature-cols{
+   display: flex;
+   position: static;
+ }
+
+ .feature-col-l{
+   max-width:50%
+ }
+ .feature-col-r{
+   background-color: rgba(224, 99, 58, 0.1);
+   max-width: 60%;
+
+   text-align: center;
+   /*padding: 300px auto 0;*/
+ }
+
+ .feature-text{
+   /*text-align: center;*/
+   margin-top: 120px;
+   font-size: 30px;
+ }
```

Comment:
Trick for the orange quotation marks: you can use the [`:before` and `:after` pseudo elements](https://developer.mozilla.org/en-US/docs/Web/CSS/::before) on your `feature-text` class.